erpnext/education/doctype/student_group/student_group.js

```diff
@@ -1,6 +1,9 @@
-cur_frm.add_fetch('student', 'title', 'student_name');
 
-frappe.ui.form.on('Student Group', {
+frappe.ui.form.on('Student Group', { 
+	setup: function(frm) { 
+		frm.add_fetch('student', 'title', 'student_name');
+	}, 
```

Comment:
This can be moved to **Fetch From** in the DocType specification of **Student Group** DocType.
Essentially no js code is required to make this work. You can remove `add_fetch` from here and drive as mentioned above.

Note: We do not encourage using `add_fetch` in the develop branch (version 13 branch). It is actively used only on the previous versions (v12, v11, etc.)

Reply:
I see.  Thanks a lot for that.  Not sure if it has already been updated on the json of the doctype.  Otherwise, i'll try to resubmit another PR.  
I will then close this.